src/Explorer/Controls/Settings/SettingsSubComponents/ThroughputInputComponents/ThroughputInputAutoPilotV3Component.tsx

```diff
@@ -620,13 +623,20 @@ export class ThroughputInputAutoPilotV3Component extends React.Component<
     if (this.IsComponentDirty().isDiscardable) {
       warningMessage = saveThroughputWarningMessage;
     }
+    const classNames = mergeStyleSets({
+      orange: [{ color: "orange" }],
+    });
 
     return (
       <>
         {warningMessage && (
-          <MessageBar messageBarIconProps={{ iconName: "WarningSolid", className: "messageBarWarningIcon" }}>
+          // <MessageBar messageBarIconProps={{ iconName: "WarningSolid", className: "messageBarWarningIcon" }}>
```

Comment:
What was the issue with using `MessageBar` component? We should try to use fluent UI components as much as possible.

Also please remove commented out code if they are no longer used.